internal/impl/redis/output_streams.go

```diff
@@ -137,7 +142,7 @@ func (r *redisStreamsWriter) WriteWithContext(ctx context.Context, msg *message.
 	if msg.Len() == 1 {
 		if err := client.XAdd(&redis.XAddArgs{
 			ID:           "*",
-			Stream:       r.conf.Stream,
+			Stream:       r.interpolatedStreamName.String(0, msg),
```

Comment:
This magic value (index `0`) is due to the fact that there is a single message in the batch. Is it obvious enough and have I understood the message loop for the expression handling correctly?